dc-cudami-server/dc-cudami-server-business/src/main/java/de/digitalcollections/cudami/server/business/impl/service/identifiable/entity/EntityServiceImpl.java

```diff
@@ -60,6 +94,23 @@ public E getByRefId(long refId) {
     return ((EntityRepository<E>) repository).getRelatedFileResources(entityUuid);
   }
 
+  @Override
+  public E save(E entity) throws IdentifiableServiceException {
+    try {
+      E entityDb = super.save(entity);
+      Optional<String> hook =
+          hookProperties.getHookForActionAndType("save", entityDb.getEntityType());
+      if (hook.isPresent()) {
+        URI url = buildNotificationUrl(hook.get(), entityDb.getUuid(), entityDb.getEntityType());
+        sendNotification("POST", url);
+      }
```

Comment:
in eine methode auslagern und "save", "POST" als parameter übergeben (und bei update wiederverwenden)

Reply:
oder besser: gleich sendNotification um fehlendes argument "action" (="save") erweitern.